aidial_assistant/commands/open_api.py

```diff
@@ -8,22 +8,35 @@
     ExecutionCallback,
     ResultObject,
 )
-from aidial_assistant.open_api.requester import OpenAPIEndpointRequester
+from aidial_assistant.open_api.requester import (
+    OpenAPIEndpointRequester,
+    ParamMapping,
+)
 
 
 class OpenAPIChatCommand(Command):
```

Comment:
It has nothing to do with Chat strictly speaking. Maybe simply `OpenAPICommand`?